### PR TITLE
Implement remaining cards from Oberyn's Revenge

### PR DESCRIPTION
**Pull request by DukeTax**

### CODE REVIEW

File: server/game/player.js [hunk 1]
```diff
@@ -896,6 +896,19 @@ class Player extends Spectator {
         });
     }
 
+    moveCardToBottomOfDeck(card, allowSave = true) {
```
Comment (cryogen) [inline server/game/player.js]: I suspect @ystros might suggest that this work similarly to the above return to hand method and to factor out some of this common code

Comment (ystros) [inline server/game/player.js]: It probably should be wrapped in a `allowGameAction` for immunity purposes, like return to hand.

I would say hold off on factoring out a common method, though. Dupes + saves in general need to be reworked for issues like #1231 and #1024, which should probably be its own PR.
